logmerge.py

```diff
@@ -215,20 +213,21 @@ def process(paths,
             wrap=None,                # Wrap long lines to this many chars.
             wrap_indent=None,         # Indentation of wrapped secondary lines.
             w=None,                   # Optional output stream.
+            verbosity=0,              # verbosity.
             bar=None):                # Optional progress bar.
     # Find log files.
     paths, total_size, path_sizes = expand_paths(paths, suffix)
 
     if not path_prefix:
         if len(paths) > 1:
-            path_prefix = os.path.commonprefix(paths)
+            path_prefix = paths[0].split(':')
```

Comment:
little worried about this, as i think commonprefix did provide some value here, which we might lose with a switch to ':' splitting

Reply:
for example, I'd worry that non-zip file input might be affected here?

Reply:
OK, will recheck how better to use that. And other containers could be a feature request!